apps/meteor/tests/end-to-end/api/livechat/14-units.ts

```diff
@@ -409,4 +413,428 @@ import { IS_EE } from '../../../e2e/config/constants';
 			await deleteUser(user);
 		});
 	});
+
+	const testDepartmentsInUnit = (unitId: string, unitName: string, numDepartments: number) => {
```

Comment:
It would be better to have an object as a parameter here, mainly to avoid magic numbers when calling the function: `testDepartmentsInUnit('test', 'test', 0)`